cmake/YarpFindDependencies.cmake

```diff
@@ -520,6 +520,8 @@ yarp_dependent_option(YARP_COMPILE_yarpmotorgui "Do you want to compile yarpmoto
                       "YARP_COMPILE_EXECUTABLES;YARP_COMPILE_GUIS;YARP_HAS_Qt5" OFF)
 yarp_dependent_option(YARP_COMPILE_yarpbatterygui "Do you want to compile yarpbatterygui?" ON
                       "YARP_COMPILE_EXECUTABLES;YARP_COMPILE_GUIS;YARP_HAS_Qt5;Qt5_VERSION VERSION_GREATER_EQUAL 5.4;" OFF)
+yarp_dependent_option(YARP_COMPILE_yarpmobilebasegui "Do you want to compile yarpmobilebasegui?" ON
+                      "YARP_COMPILE_EXECUTABLES;YARP_COMPILE_GUIS;YARP_HAS_Qt5;Qt5_VERSION VERSION_GREATER_EQUAL 5.4;" OFF)
```

Comment:
Are you actually using any feature that require Qt 5.4 or greater, or is this a copy-paste from the 2 lines above?